index.html

```diff
@@ -13,30 +13,197 @@
+          required.</p>
+        <ul class="language-categories">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">Javascript</a></li>
+        </ul>
+        <button class="submit-btn" type="submit">See Project</button>
+      </div>
     </div>
-    <div class="social">
-      <h3>LET'S CONNECT</h3>
-      <ul class="media-icons">
-        <li><a><img src="Assets/images/Twitter.png" alt="twitter"></a></li>
-        <li><a><img src="Assets/images/Linkedin icon.png" alt="linkedin"></a></li>
-        <li><a><img src="Assets/images/Mail.png" alt="mail"></a></li>
-        <li><a><img src="Assets/images/Github.png" alt="github"></a></li>
-        <li><a><img src="Assets/images/peace.png" alt="vector"></a></li>
-      </ul>
+   </section>
+  
+   <!----About Sectiom------->
+  
+  <section class="container-about">
+    <div class="card-about">
+      <div>
+        <h3 class="about-project-title">About Myself</h3>
+        <p class="description-about">Hello I’m a software developer! I can help you build a product , feature or website
+          Look through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.</p>
+      </div>
+      <div class="social">
+        <h3>LET'S CONNECT</h3>
+        <ul class="media-icons">
+          <li><a><img src="Assets/images/Github.png" alt="github"></a></li>
+          <li><a><img src="Assets/images/Linkedin icon.png" alt="linkedin"></a></li>
+          <li><a><img src="Assets/images/peace.png" alt="peace"></a></li>
+          <li><a><img src="Assets/images/Twitter.png" alt="twitter"></a></li>
+          <li><a><img src="Assets/images/Mail.png" alt="mail"></a></li>
+        </ul>
+      </div>
+      <button class="submit-btn" type="submit">Get my resume</button>
+      <div>
+        <ul class="skill-list">
+          <li>
+            <p class="list"><span>Language</span> <i class="fa-solid fa-chevron-down"></i></p>
+              <ul class="lang">
+                <li><a href="#"><img src="Assets/images/js.svg" alt=""> JavaScript</a></li>
+                <li><a href="#"> <img src="Assets/images/html.svg" alt=""> HTML</a></li>
+                <li><a href="#"><img src="Assets/images/css.svg" alt=""> css</a></li>
+              </ul>
+          </li>
+          <li>
+            <p class="list"><span>Frameworks</span> <i class="fa-solid fa-chevron-right"></i></p>
+          </li>
+          <hr>
+          <li>
+            <p class="list"><span>Skills</span> <i class="fa-solid fa-chevron-right"></i></p>
+          </li>
+          <hr>
+        </ul>
+      </div>
     </div>
   </section>
+  
+  <!-------Contact Section----->
+  
+  <section class="contact-me">
+    <div class="container">
+  
+      <!-- contact text -->
+  
+      <div class="footer-text">
+        <h1>Contact me</h1>
```

Comment:
- Kindly, you will have only one `h1` tag par page for semantic reasons, so use other heading tags for contact me `h2-h6` 👍